core/arch/arm/plat-rockchip/platform_rk3566.c

```diff
@@ -0,0 +1,44 @@
+#include <common.h>
```

Comment:
This file looks like a copy of `platform_rk3399.c`. I'd rather reuse the same code than use diverging copies that are poorly understood.

Reply:
Sure, but firstly I need to get `SGRF` info from Rockchip.

Reply:
Unfortunately, RockChip will not provide any SGRF to anyone who is not a $$$ customer. I have a somewhat decent relationship with them, I even reported to them a critical security vulnerability with one of their SoCs and they still won't help much, even signing NDAs.